stable_baselines/common/cmd_util.py

```diff
@@ -14,20 +15,89 @@
 from stable_baselines.common.vec_env import DummyVecEnv, SubprocVecEnv
 
 
+def make_vec_env(env_id, n_envs=1, seed=None, start_index=0,
+                 use_subprocess=False, start_method=None,
+                 monitor_dir=None, wrapper_class=None,
+                 env_kwargs=None, vec_env_cls=None, vec_env_kwargs=None):
+    """
+    Create a wrapped, monitored VecEnv.
+
+    :param env_id: (str or Type[gym.Env]) the environment ID or the environment class
+    :param n_envs: (int) the number of environments you wish to have in parallel
+    :param seed: (int) the inital seed for the random number generator
+    :param start_index: (int) start rank index
+    :param start_method: (str) method used to start the subprocesses.
+        See SubprocVecEnv doc for more information
+    :param use_subprocess: (bool) Whether to use `SubprocVecEnv` or `DummyVecEnv`,
```

Comment:
Do we really need this and `vec_env_cls`? Is it that much harder for someone to write `vec_env_cls=SubprocVecEnv` than `is_subprocess=True`? I'm inclined to get rid of it, but don't strongly object to its presence.

Reply:
This is more for convenience, as 99% of the time, you are going to use `DummyVecEnv` or `SubprocVecEnv`, same for `start_method` argument. This remove the need of one additional import.
But I don't have a strong view on that neither as I'm using `DummyVecEnv` 100% of the time ^^

Reply:
I vote for removing `is_subprocess` as it could cause some confusion like this (e.g. what happens if you give custom VecEnv class and `is_subprocess`?). While admittedly `is_subprocess` is more straight-forward, I do not think it is too much to ask to set custom VecEnv to subprocess-one via the argument we already have.

Reply:
now fixed ;)